src/commands/hex.ts

```diff
@@ -0,0 +1,37 @@
+import { Command } from "./command";
+import { hexes, Hex } from "../data/hexes";
+
+const hexExpressionMap = hexes.reduce(
```

Comment:
Typing the starting object is one way to accomplish typing this, the other is you can do `hexes.reduce<{ [key: string]: Hex}>()` as well